README.md

```diff
@@ -52,21 +52,13 @@ You can define your own network in Bender using our custom operator or you can l
 ```swift
 import MetalBender
 
-// Define a network and how it will load its weights / parameters
-let randomLoader = RandomParameterLoader(maxSize: 7*7*64*1024)
-let network = Network(device: device, inputSize: LayerSize(f: 3, w: 256), parameterLoader: randomLoader)
-
-// Convert a graph from TensorFlow
-let url = Bundle.main.url(forResource: "myGraph", withExtension: "pb")!
-let converter = TFConverter.default()
-network.convert(converter: converter, url: url, type: .binary)
-
-// Initialize the network
-network.initialize()
+// Load a model from TensorFlow
+let url = Bundle.main.url(forResource: "myModel", withExtension: "pb")!
+let network = Network.load(url: url, converter: TFConverter.default(), inputSize: LayerSize(f: 3, w: 256))
 
 // Run the network
-let inputImage = ...
-network.run(inputImage: image, queue: commandQueue) { output in
+let input = ...
```

Comment:
What about removing this line and placing the ellipsis when passing the parameter? And maybe we should comment the ellipsis like `// ...` or `/* ... */` here and in other places.